articles/cognitive-services/Face/QuickStarts/Python.md

```diff
@@ -4,317 +4,141 @@ description: Get information and code samples to help you quickly get started us
-body = "{'url':'https://upload.wikimedia.org/wikipedia/commons/c/c3/RH_Louise_Lillian_Gish.jpg'}"
-
-try:
-    # Execute the REST API call and get the response.
-    conn = httplib.HTTPSConnection('westcentralus.api.cognitive.microsoft.com')
-    conn.request("POST", "/face/v1.0/detect?%s" % params, body, headers)
-    response = conn.getresponse()
-    data = response.read()
-
-    # 'data' contains the JSON data. The following formats the JSON data for display.
-    parsed = json.loads(data)
-    print ("Response:")
-    print (json.dumps(parsed, sort_keys=True, indent=2))
-    conn.close()
-
-except Exception as e:
-    print("[Errno {0}] {1}".format(e.errno, e.strerror))
-
-####################################
-
-########### Python 3.6 #############
-import http.client, urllib.request, urllib.parse, urllib.error, base64, requests, json
-
-###############################################
-#### Update or verify the following values. ###
-###############################################
-
-# Replace the subscription_key string value with your valid subscription key.
-subscription_key = '13hc77781f7e4b19b5fcdd72a8df7156'
-
-# Replace or verify the region.
-#
-# You must use the same region in your REST API call as you used to obtain your subscription keys.
-# For example, if you obtained your subscription keys from the westus region, replace 
-# "westcentralus" in the URI below with "westus".
-#
-# NOTE: Free trial subscription keys are generated in the westcentralus region, so if you are using
-# a free trial subscription key, you should not need to change this region.
-uri_base = 'https://westcentralus.api.cognitive.microsoft.com'
-
-# Request headers.
-headers = {
-    'Content-Type': 'application/json',
-    'Ocp-Apim-Subscription-Key': subscription_key,
-}
 
-# Request parameters.
+```python
+import requests
+from IPython.display import HTML
+
+headers = { 'Ocp-Apim-Subscription-Key': subscription_key }
+    
 params = {
     'returnFaceId': 'true',
     'returnFaceLandmarks': 'false',
     'returnFaceAttributes': 'age,gender,headPose,smile,facialHair,glasses,emotion,hair,makeup,occlusion,accessories,blur,exposure,noise',
 }
 
-# Body. The URL of a JPEG image to analyze.
-body = {'url': 'https://upload.wikimedia.org/wikipedia/commons/c/c3/RH_Louise_Lillian_Gish.jpg'}
+response = requests.post(face_api_url, params=params, headers=headers, json={"url": image_url})
+faces = response.json()
+HTML("<font size=5>Detected <font color='blue'>%d</font> faces in the image</font>"%len(faces))
+```
+
 
-try:
-    # Execute the REST API call and get the response.
-    response = requests.request('POST', uri_base + '/face/v1.0/detect', json=body, data=None, headers=headers, params=params)
 
-    print ('Response:')
-    parsed = json.loads(response.text)
-    print (json.dumps(parsed, sort_keys=True, indent=2))
 
-except Exception as e:
-    print('Error:')
-    print(e)
+<font size=5>Detected <font color='blue'>2</font> faces in the image</font>
```

Comment:
Why having this line here?

Reply:
Expected.. it's the output of the input image.